source/amazon/services/supported-services/cloudwatchlogs.rst

```diff
@@ -0,0 +1,76 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _aws_cloudwatchlogs:
+
+AWS CloudWatch Logs
+===================
+
+.. versionadded:: 4.0.0
+
+`AWS CloudWatch Logs <https://aws.amazon.com/cloudwatch/>`_ is a service that enables the user to centralize the logs from all his systems, applications and AWS services in a single place. CloudWatch Logs works using log groups, log streams and log events:
```

Comment:
I would use `its` or `their` instead of "enables the user to centralize the logs from all **his** systems" to avoid using genders.

Reply:
Changed to `is a service that enables the users to centralize the logs from all their systems, applications and AWS services in a single place'`